src/containers/ProjectDashboardContainer.jsx

```diff
@@ -32,7 +32,7 @@ class ProjectDashboardContainer extends Component {
     return (
       <div>
         <h2>Project Dashboard</h2>
-        {React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
+        {project.primary_language && React.cloneElement(this.props.children, { fieldguides, pages, project, workflows, tutorials })}
```

Comment:
And maybe moving some actions to the top-level container. I need to have a look through the component tree to see what's being passed down. `project` might already be there.